app/services/i_scream_service.rb

```diff
@@ -1,8 +1,6 @@
 class IScreamService 
 
   def conn 
-    # TO RUN LOCALLY, SWAP LINES 5 and 6
-    # Faraday.new('http://localhost:9393') do |faraday|
```

Comment:
removed as many comments as I could find. I left the 'keyword' methods commented out for demo comp